packages/app/src/components/table/cell/string/index.tsx

```diff
@@ -10,6 +10,17 @@ export type Props = BaseProps & {
 const CellForTypeString: React.FC<Props> = ({ on, schema, value }) => {
   const content = useMemo<JSX.Element>(() => {
     switch (schema.format) {
+      case 'uri':
+        return (
+          <a
```

Comment:
This `<a>` should be set to "stopPropagation".
However, I have determined that this problem is not fatal and will fix it in the next release.